src/accountant/core.cljs

```diff
@@ -141,7 +141,7 @@
                          (str route "?" query-string))]
        (if (= old-route route)
```

Comment:
Could we just ignore old/new comparison and just always use set-token! instead? (do not know much about the trade offs involved, so advice is appreciated :))

My use-case:
While at the same page, would like to update query parameters based on user input

Reply:
@mbutlerw  Tested it and it fixes my problem. Could you consider this suggestion?

```
(if (= old-route route)
           (. history (replaceToken with-params))
           (set-token! history with-params nil))
```
=>

```
(set-token! history with-params nil)
```